openmc/filter_expansion.py

```diff
@@ -463,3 +463,65 @@ def to_xml_element(self):
         subelement.text = str(self.r)
 
         return element
+
+
+class ZernikeRadialFilter(ZernikeFilter):
+    r"""Score radial Zernike expansion moments in space up to specified order.
+
+    The Zernike polynomials are defined the same as in ZernikeFilter.
```

Comment:
Instead of saying `ZernikeFilter`, use ``class:`ZernikeFilter` `` so a hyperlink will be made in the docs.